src/withESI.tsx

```diff
@@ -48,11 +50,11 @@ export default function withESI<P>(
       this.esi = esi || {};
       this.state.childProps = childProps;
 
-      if (!(process as IWebpackProcess).browser) {
+      if (typeof window === "undefined") {
```

Comment:
I've had problems with that, but I can't remember exactly what.

Edit: ok, problem fixed now https://stackoverflow.com/a/59562136/1352334